UI/Dialogs/BaseDialog.cpp

```diff
@@ -1086,4 +1116,85 @@ namespace dialog
 	{
 		return m_lpMapiObjects;
 	}
+	
+	std::wstring LookupFamilyName(_In_ PCWSTR familyName, _In_ const UINT32 filter)
+	{
+		UINT32 count = 0;
+		UINT32 length = 0;
+		std::wstring buildInfo;
+		LONG rc = FindPackagesByPackageFamily(familyName, filter, &count, nullptr, &length, nullptr, nullptr);
+		if (rc == ERROR_SUCCESS)
+		{
+			output::DebugPrint(1, L"FindPackagesByPackageFamily -> No packages found\n");
```

Comment:
Don't use magic constant of 1 - should be a DBG macro you can use. Also, consider using DebugPrintEx instead, like this:
```
output::DebugPrintEx(DBGGeneric, CLASS, L"LookupFamilyName", L"My cool comment %d\n", param1);

```

Reply:
Done.